API/models/validateEntity.go

```diff
@@ -175,7 +177,32 @@ func validateJsonSchema(entity int, t map[string]interface{}) (bool, *u.Error) {
 			println(v.GoString())
 			// Format errors array
 			errSlice := []string{}
+			// Open and unmarshall types.json
```

Comment:
Maybe it's not necessary to open a file and unmarshal it every time there's an error. The schemas for example are not opened and processed every time we are going to validate something (check init)

Reply:
That's right! I've modified the initialisation so that a map with all descriptions is created while loading the schemas, thus opening the file and processing the data only once, instead of every time `validateJsonSchema()` is called. Thanks!